telebot/types.py

```diff
@@ -2432,23 +2432,29 @@ class InlineKeyboardMarkup(Dictionaryable, JsonSerializable, JsonDeserializable)
     This object represents an inline keyboard that appears right next to the message it belongs to.
 
     .. note::
-        It is recommended to use :meth:`telebot.service_utils..quick_markup` instead.
+        It is recommended to use :meth:`telebot.util.quick_markup` instead.
 
     .. code-block:: python3
         :caption: Example of a custom keyboard with buttons.
 
-        from telebot.service_utils..import quick_markup
+        from telebot.util import quick_markup
 
-        markup = quick_markup(
-            {'text': 'Press me', 'callback_data': 'press'},
-            {'text': 'Press me too', 'callback_data': 'press_too'}
-        )
+        markup = quick_markup({
+            'Twitter': {'url': 'https://twitter.com'},
+            'Facebook': {'url': 'https://facebook.com'},
+            'Back': {'callback_data': 'whatever'}
+        }, row_width=2)
+        # returns an InlineKeyboardMarkup with two buttons in a row, one leading to Twitter, the other to facebook
+        # and a back button below
 
     Telegram Documentation: https://core.telegram.org/bots/api#inlinekeyboardmarkup
 
-    :param inline_keyboard: :obj:`list` of button rows, each represented by an :obj:`list` of 
+    :param keyboard: :obj:`list` of button rows, each represented by an :obj:`list` of 
```

Comment:
@Badiboy [doc](https://core.telegram.org/bots/api#inlinekeyboardmarkup) says that it should be inline_keyboard. Ig we should have another PR changing keyboard param to inline_keyboard

Reply:
Do I need to change this back to `inline_keyboard`? However, I do notice other inconsistencies of parameter names between the library and the Telegram API documentation. For example, the `InlineQueryResultPhoto` class has parameter `thumb_url`, while the name of the field in the API documentation is `thumbnail_url` instead.

Reply:
> @Badiboy [doc](https://core.telegram.org/bots/api#inlinekeyboardmarkup) says that it should be inline_keyboard.

1. He just align the **description** with **actual name**.
![image](https://user-images.githubusercontent.com/5613421/228057636-bf9af7fa-9ffa-4a7f-8f5a-6fe1a33ae608.png)
And this is correct update.

2. doc have nothing about **class constructor parameters**. It describes **fields**. So construtor may have parameter "keyboard" and it is more laconic then "inline_keyboard". And "inline_" prefix is absolete because you create InlineKeyboard: it's absolutely clear that keyboard is inline.

3. Anyway it's not possible to change keyboard to inline_keyboard due to compatibility. And I see no vvaluable reason to create new parameter and deprecate old.